mathics/builtin/assignments/assignment.py

```diff
@@ -273,14 +273,14 @@ class TagSet(Builtin, _SetOperator):
     </dl>
 
     Create an upvalue without using 'UpSet':
-    >> x /: f[x] = 2
-     = 2
-    >> f[x]
-     = 2
-    >> DownValues[f]
+    >> square /: area[square] = s^2
```

Comment:
IMHO, this is better:
>> square /: area[square[s_]] := s^2

Reply:
I trust your judgement in these things. Would you commit over this and improve the examples? 

If I try to do that, I am sure that I will get something wrong.

Reply:
OK, since this change seems to be completely unrelated to the rest, I can change this after merge.